specification/batch/data-plane/Microsoft.Batch/2017-09-01.6.0/BatchService.json

```diff
@@ -11300,6 +11333,11 @@
           "title": "The command line of the Job Release task.",
           "description": "The command line does not run under a shell, and therefore cannot take advantage of shell features such as environment variable expansion. If you want to take advantage of such features, you should invoke the shell in the command line, for example using \"cmd /c MyCommand\" in Windows or \"/bin/sh -c MyCommand\" in Linux."
         },
+        "containerSettings": {
+          "$ref": "#/definitions/TaskContainerSettings",
+          "title": "The settings for the container under which the Job Release task runs.",
+          "description": "When this is specified, all directories recursively below the AZ_BATCH_NODE_ROOT_DIR (the root of batch directories on the node) are mapped into the container, all task enviornment variables are mapped into the container, and the task command line is executed in the container."
```

Comment:
Should we warn people that if they have JP/JR in containers then they will be isolated from each other?  Perhaps more something for the article.

Reply:
I think it makes more sense to discuss that in the article. The description here is relatively long as is, and I think that "it runs in a container" sort of implies "stuff you do in that container doesn't persist to the host or other containers"